javascript/javascript1/week3/homework4.js

```diff
@@ -0,0 +1,74 @@
+// Smart-ease - Back to the basics!
+// NOnoN0nOYes (Note taking app)
+
+// The saveNote function should push an object to the notes array with the keys content and id.
+
+const notes = [];
+/**
+ *
+ *
+ * @param {string} content
+ * @param {number} id
+ */
+
+function saveNote(content, id) {
+  notes.push({
+    content: content,
+    id: id,
+  });
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+
+//unique feature of colored log
+console.log("\x1b[31m%s\x1b[0m", "\n\t SaveNote task");
```

Comment:
Didn't work in the terminal of CodePen, where I was testing your code. Could you share a link to where you found it?

Reply:
it is called string substitution with %s  and ansi escape codes: https://en.wikipedia.org/wiki/ANSI_escape_code#Colors. , maybe it works with "\x1b[31m%s"

Reply:
Cool. Tried it in VC Code - it works. Thanks, I learned something new.